pkg/daemon/ceph/client/filesystem.go

```diff
@@ -93,6 +96,19 @@ func ListFilesystems(context *clusterd.Context, clusterName string) ([]CephFiles
 	return filesystems, nil
 }
 
+func GenerateMetadataPoolName(fsName string, metadataPool *model.Pool) *model.Pool {
```

Comment:
exported function GenerateMetadataPoolName should have comment or be unexported

Reply:
the comment must start by the name of the function.

Reply:
I have changed the comments and now they start with the name of the functions and also I have added the commit message describing what my PR is actually doing.
Thanks again Seb!

Reply:
It's not a method but a function since it has no receiver

Reply:
This should have a fuller description as well. In my words, this is a helper method that generates a new name for the given metadata pool and returns the renamed pool.